src/contractTest/java/uk/gov/hmcts/reform/lrdapi/provider/LrdApiProviderTest.java

```diff
@@ -86,8 +97,9 @@ void pactVerificationTestTemplate(PactVerificationContext context) {
     @BeforeEach
     void before(PactVerificationContext context) {
         MockMvcTestTarget testTarget = new MockMvcTestTarget();
+        System.setProperty("pact.verifier.publishResults", "true");
         testTarget.setControllers(
-            lrdApiController);
+            lrdApiController, lrdCourtVenueController);
         if (context != null) {
             context.setTarget(testTarget);
```

Comment:
use non null or is null avoid using null

Reply:
Addressed in bd3f316